### PR TITLE
Add linting to Docker build

### PR DESCRIPTION
**Pull request by moz-astults**
Adding npm run lint to Dockerfile so we can start enforcing a successful Docker build (and lint) as a prerequisite to merge. Working with @maxxcrawford to have linting pass.

### CODE REVIEW

**Comment by moz-astults** [comment]
From the CircleCI build showing a failed lint:
```
 ---> aa8700f49143
Removing intermediate container 5890fd1353bc
Step 8/24 : RUN npm run lint
 ---> Running in f760d2cbe7fc

> fx-private-relay@1.0.0 lint /app
> npm run lint:js && npm run lint:css


> fx-private-relay@1.0.0 lint:js /app
> eslint .


Oops! Something went wrong! :(

ESLint: 7.30.0

ESLint couldn't find a configuration file. To set up a configuration file for this project, please run:

    eslint --init

ESLint looked for configuration files in /app and its ancestors. If it found none, it then looked in your home directory.

If you think you already have a configuration file or if you need more help, please stop by the ESLint chat room: https://eslint.org/chat/help

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! fx-private-relay@1.0.0 lint:js: `eslint .`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the fx-private-relay@1.0.0 lint:js script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-07-08T20_39_20_407Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! fx-private-relay@1.0.0 lint: `npm run lint:js && npm run lint:css`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the fx-private-relay@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-07-08T20_39_20_424Z-debug.log
The command '/bin/sh -c npm run lint' returned a non-zero code: 2

Exited with code exit status 2

CircleCI received exit code 2
```

**Review by maxxcrawford — APPROVED** [review]

File: static/js/app.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ async function sendForm(formAction, formData) {
 }
 
 function copyToClipboardAndShowMessage(triggeringEl) {
-  const aliasCopyBtn = triggeringEl;
+  // const aliasCopyBtn = triggeringEl;
```
Comment (groovecoder) [inline static/js/app.js:92]: Should we just remove this line?
```suggestion
```

File: static/js/app.js [hunk 2]
```diff
@@ -260,7 +260,7 @@ function addEventListeners() {
   const mobileMenuWrapper = document.querySelector(".mobile-menu");
   if (mobileMenuWrapper) {
     const mobileMenuButton = document.querySelector(".mobile-menu-toggle");
-    const mobileMenuLinks = document.querySelector(".mobile-menu-links");
+    // const mobileMenuLinks = document.querySelector(".mobile-menu-links");
```
Comment (groovecoder) [inline static/js/app.js:263]:
```suggestion
```

**Review by groovecoder — CHANGES_REQUESTED** [review]
LGTM. Just remove the 2 commented lines.

File: static/js/app.js (hunk 1 above)
Comment (maxxcrawford) [reply static/js/app.js:92]: @groovecoder Dang! I should've commented here. Neither this (or the comment) below was actually in use. I think we can remove them. I just didn't wanna mess anything up.

File: static/js/app.js [hunk 3]
```diff
@@ -89,7 +89,6 @@ async function sendForm(formAction, formData) {
 }
 
 function copyToClipboardAndShowMessage(triggeringEl) {
-  const aliasCopyBtn = triggeringEl;
```
Comment (maxxcrawford) [inline static/js/app.js:92]: @groovecoder Removed (and rebased)

**Review by groovecoder — APPROVED** [review]

**Pull request merged**